lib/cache.js

```diff
@@ -28,6 +29,7 @@ module.exports = class Cache {
 
     this.latest = {}
     this.lastUpdate = null
+    this.latestReleaseByEnvironment = {}
```

Comment:
They used to only track the latest release, but now we track the latest release keyed on env name.